app/Models/Event.php

```diff
@@ -275,10 +275,7 @@ public function parseMatchesAchievement($matches)
         }
 
         return [
-            'achievement' => [
-                'slug' => $achievement->slug,
-                'name' => $achievement->name,
-            ],
+            'achievement' => json_item($achievement, 'Achievement'),
```

Comment:
is the second field just for display purposes? if so, medal is probably better

```suggestion
            'achievement' => json_item($achievement, 'Medal'),
```